d3a_interface/constants_limits.py

```diff
@@ -46,72 +47,66 @@ class GeneralSettings:
         # Boolean flag which forces d3a to dispatch events via redis channels
         EVENT_DISPATCHING_VIA_REDIS = False
 
-        RATE_DECREASE_PER_UPDATE_RANGE = RangeLimit(0, 1000)
-        RATE_CHANGE_PER_UPDATE = RangeLimit(0, 1000)
+        RATE_CHANGE_PER_UPDATE_LIMIT = RangeLimit(0, 1000)
 
-        INITIAL_RATE_OPTIONS = [1, 2, 3]
-
-        NUM_CLONES_RANGE = RangeLimit(0, 100)
+        NUM_CLONES_LIMIT = RangeLimit(0, 100)
         MIN_NUM_TICKS = 10
         MIN_SLOT_LENGTH_M = 2
         MAX_SLOT_LENGTH_M = 60
         MIN_TICK_LENGTH_S = 1
 
     class CommercialProducerSettings:
-        ENERGY_RATE_RANGE = RangeLimit(0, 10000)
-        MAX_POWER_KW_RANGE = RangeLimit(0, 10000000)
+        ENERGY_RATE_LIMIT = RangeLimit(0, 10000)
+        MAX_POWER_KW_LIMIT = RangeLimit(0, 10000000)
 
     class StorageSettings:
         # least possible state of charge
-        MIN_SOC_RANGE = RangeLimit(10, 99)
+        MIN_SOC_LIMIT = RangeLimit(10, 99)
         # possible range of state of charge
-        INITIAL_CHARGE_RANGE = RangeLimit(10, 100)
+        INITIAL_CHARGE_LIMIT = RangeLimit(10, 100)
 
-        BREAK_EVEN_RANGE = RangeLimit(0, 10000)
         # Max battery capacity in kWh.
         CAPACITY = 1.2
-        CAPACITY_RANGE = RangeLimit(0.0001, 2000000)
+        CAPACITY_LIMIT = RangeLimit(0.0001, 2000000)
         # Maximum battery power for supply/demand, in Watts.
         MAX_ABS_POWER = 5
-        MAX_ABS_POWER_RANGE = RangeLimit(0.0001, 2000000)
-        # Energy buy-range, storage never buys outside this limit.
+        MAX_ABS_POWER_LIMIT = RangeLimit(0.0001, 2000000)
+        # Energy buy-limit, storage never buys outside this limit.
```

Comment:
Minor, but this is a range (of course it is a limit somwhow, but in order to not get confused in the future, please change it to range again. Thanks!)

Reply:
Okay, should it still use `RangeLimit` as a namedtuple or `RateRange`

Reply:
yes of course!